exercises/crypto-square/description.md

```diff
@@ -60,6 +62,6 @@ mayoogo
 anouuio
 ntnnlvt
 wttddes
-aohghn
-sseoau
+aohghns
```

Comment:
now reading the text top to bottom, it will be misaligned - before this change it was "if man was meant to stay on the ground god would have given us roots", and now it is "if man was meant to etay on tho ground gad would huve given s rootss" - is that okay? One can argue it is a measure to obscure the plain text, in which case we should just pick a group size unrelated to the square size, such as "always 5"

You can read previous discussion about it in #190, but it does appear the https://github.com/exercism/x-common/blob/master/exercises/crypto-square/canonical-data.json file matches the version *before* this PR rather than *after* it.